.github/workflows/ci.yaml

```diff
@@ -0,0 +1,36 @@
+name: CI
+
+on: [ push, pull_request ]
+
+jobs:
+  run:
+    runs-on: ${{ matrix.operating-system }}
+    strategy:
+      matrix:
+        operating-system: [ ubuntu-latest ]
+        php-versions: [ '8.0', '8.1', '8.2' ]
+    name: PHP ${{ matrix.php-versions }} Test on ${{ matrix.operating-system }}
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1
+
+      - name: Setup PHP
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: ${{ matrix.php-versions }}
+          extensions: ast
+          coverage: pcov
+
+      - name: Validate composer.json and composer.lock
+        run: composer validate
+
+      - name: Install dependencies
+        run: COMPOSER_ROOT_VERSION=0.2.5 composer install --prefer-dist --no-progress
```

Comment:
Adding the `COMPOSER_ROOT_VERSION` environment variable to resolve the following error:

```
  Problem 1
    - Root composer.json requires infection/infection ^0.19.0 -> satisfiable by infection/infection[0.19.0, 0.19.1, 0.19.2].
    - infection/infection[0.19.0, ..., 0.19.2] require infection/include-interceptor ^0.2.4 -> satisfiable by infection/include-interceptor[0.2.4, 0.2.5] from composer repo (https://repo.packagist.org) but infection/include-interceptor is the root package and cannot be modified. See https://getcomposer.org/dep-on-root for details and assistance.
```